helix-core/src/main/java/org/apache/helix/GroupCommit.java

```diff
@@ -112,6 +112,9 @@ public boolean commit(BaseDataAccessor<ZNRecord> accessor, int options, String k
             merged = accessor.get(mergedKey, null, options);
```

Comment:
I suggest covering all the accessor related calls inside the commit(). There should be 3 as I remember.

Reply:
What do you mean?